code/datalab-app/package.json

```diff
@@ -16,9 +16,10 @@
     "graphql-server-express": "1.1.2",
     "graphql-tag": "2.4.2",
     "history": "4.6.1",
+    "jquery": "^3.2.1",
```

Comment:
Why the jQuery dependency? Material doesn't depend on it does it? https://github.com/callemall/material-ui/blob/v1-beta/package.json

Is this a hangover from trying to get the dropdown to work?

If we do need it then please remove the `^`.

Reply:
Unfortunately jQuery is a missing dependency for Toastr. This is was previously covered by SemanticUI dependency.